code/datums/status_effects/neutral.dm

```diff
@@ -620,17 +620,39 @@
 	desc = "Waaater... Fuck this WATER!!"
 	icon_state = "shower_regen_catgirl"
 
+/atom/movable/screen/alert/status_effect/shower_regen/bloody_like
+	name = "Washing"
+	desc = "Mhhhmmmm... the crimson red drops of life. How delightful."
+	icon_state = "shower_regen_blood_happy"
+
+/atom/movable/screen/alert/status_effect/shower_regen/bloody_dislike
+	name = "Washing"
+	desc = "Is that... blood? What the fuck!"
+	icon_state = "shower_regen_blood_bad"
+
 /datum/status_effect/shower_regen
 	id = "shower_regen"
 	duration = -1
 	status_type = STATUS_EFFECT_UNIQUE
 	alert_type = /atom/movable/screen/alert/status_effect/shower_regen
 	/// How many heals from washing.
 	var/stamina_heal_per_tick = 4
+	/// Is this the shower raining blood?
+	var/bloody_shower = FALSE
+
+/datum/status_effect/shower_regen/on_creation(mob/living/new_owner, _bloody_shower = FALSE)
+	bloody_shower = _bloody_shower
+	return ..()
 
 /datum/status_effect/shower_regen/on_apply()
 	. = ..()
-	if(HAS_TRAIT(owner, TRAIT_WATER_HATER) && !HAS_TRAIT(owner, TRAIT_WATER_ADAPTATION))
+
+	if(bloody_shower)
+		if(HAS_TRAIT(owner, TRAIT_MORBID) || HAS_TRAIT(owner, TRAIT_EVIL))
+			alert_type = /atom/movable/screen/alert/status_effect/shower_regen/bloody_like
+		else
+			alert_type  = /atom/movable/screen/alert/status_effect/shower_regen/bloody_dislike
+	else if(HAS_TRAIT(owner, TRAIT_WATER_HATER) && !HAS_TRAIT(owner, TRAIT_WATER_ADAPTATION))
```

Comment:
So you can be a shower hater AND enjoy a bloody shower? This might need more fine tuning. Especially the regen/degen ticks in the next proc.

Reply:
It specifically says `WATER` in the trait. I also didn't really touch regen/degen stuff since I just wanted it to be mostly flavor and mood events.

Reply:
Oh I misread that if statement, nevermind.

The reason I mention the regen/degen proc is because it is checking for TRAIT_WATER_HATER but not for these other traits. The flavour here is that the former causes you to take a degen because you hate showering in water. But if you like a bloody shower, it doesn't then make sense to suffer a degen. Does that make more sense?

Reply:
Changed it to skip regen if it's a bloody shower since I'm aiming for this to be only a flavor effect.